Facebook.iOS/build.cake

```diff
@@ -1,12 +1,16 @@
 
-#load "../common.cake"
-#load "poco.ios.cake"
+#load "common.cake"
+#load "poco.cake"
 #load "components.cake"
 #load "custom_externals_download.cake"
 
 var TARGET = Argument ("t", Argument ("target", "nuget"));
 var SDKS = Argument ("sdks", "");
```

Comment:
This probably doesn't work as well anymore, but I was just wanting to get the build working. Because I split the source and samples solutions, it should be good enough for now.